CONTRIBUTING.md

```diff
@@ -0,0 +1,33 @@
+## Before getting started
+
+Thanks for your interest in contributing to **CADET-Process**!
+To streamline the development, please carefully read the following guide.
+
+# How to contribute
+
+If you need further guidance, you can find our team on the following:
+
+* [CADET-Forum](https://forum.cadet-web.de/)
+
+## Getting started
+
+> Here you should address guidelines about future functionalities, milestone labels, bugs and unlisted features, issued assignments, etc. Since now we have an `Issue` and a `Pull Request` template, you can reference them directly:
+
+For further guidance about getting started, please refer to the related links:
+
+* [Pull Request Guidelines](PULL_REQUEST_TEMPLATE.md)
```

Comment:
We should also add templates for CADET-Process (see https://github.com/modsim/CADET/pull/207)